### PR TITLE
Css title warning fixes

### PR DESCRIPTION
**Pull request by igrosso**

### CODE REVIEW

File: client/store/configureStore.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function configureStore(middlewares, initialState = { }) {
   // Enable Webpack hot module replacement for reducers.
   if (process.env.NODE_ENV !== 'production' && module.hot) {
     module.hot.accept('../reducers', () => {
-      const nextRootReducer = require('../reducers'); // eslint-disable-line global-require
+      const nextRootReducer = require('../reducers');
```
Comment (zxan1285) [inline client/store/configureStore.js]: I guess, we should keep this eslint comment

File: client/actions/script.js [hunk 2]
```diff
@@ -15,11 +15,17 @@ export function fetchScript(name) {
   };
 }
 
-export function updateScript(script, data) {
+export function updateScript(script, data, onSuccess) {
```
Comment (sandrinodimattia) [inline client/actions/script.js]: Undo the changes in this file.

File: client/app.js [hunk 3]
```diff
@@ -25,28 +25,26 @@ const reduxHistory = syncHistoryWithStore(history, store);
 
 store.subscribe(() => {
   switch (store.getState().lastAction.type) {
```
Comment (sandrinodimattia) [inline client/app.js]: Undo the changes in this file. I already fixed this.

File: client/components/Dashboard/LoadingPanel.jsx [hunk 4]
```diff
@@ -54,7 +54,6 @@ class LoadingPanel extends Component {
   componentWillReceiveProps(nextProps) {
     if (!nextProps.show) {
       clearTimeout(this.showTimer);
```
Comment (sandrinodimattia) [inline client/components/Dashboard/LoadingPanel.jsx]: Undo the changes in this file. I already fixed this.

File: client/components/Users/UserDevices.jsx [hunk 5]
```diff
@@ -32,13 +42,13 @@ export default class UserDevices extends Component {
...
           </TableHeader>
           <TableBody>
-          {Object.keys(devices).sort().map((device) =>
-            <TableRow key={device}>
-              <TableIconCell color="green" icon="243" />
-              <TableTextCell>{device}</TableTextCell>
-              <TableTextCell>{devices[device]}</TableTextCell>
-            </TableRow>
-          )}
+            {Object.keys(devices).sort().map((device) =>
```
Comment (sandrinodimattia) [inline client/components/Users/UserDevices.jsx]: Is this indenting ok? No eslint error?

File: client/components/Users/UserForm.jsx [hunk 6]
```diff
@@ -15,22 +15,22 @@ export default createForm('user', class extends Component {
...
+    appSettings: PropTypes.object
   }
 
   constructor(props) {
     super(props);
     this.state = {
       usernameRequired: false,
       memberships: false,
-      customErrors: { }
+      customErrors: {}
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Should this not have a space?

File: client/components/Users/UserForm.jsx [hunk 7]
```diff
@@ -102,51 +103,53 @@ export default createForm('user', class extends Component {
...
+      validationErrors, memberships, title, show,
+      confirmLoading, userCreateError, settings
+    } = this.props;
     const options = this.getOptions(memberships);
+    const label = settings.settings && settings.settings.dict && settings.settings.dict.memberships ? settings.settings.dict.memberships : 'Memberships';
     return (
-      <Confirm title={title} show={show} loading={confirmLoading} onCancel={this.props.hideConfirmWindow} onConfirm={this.onConfirmUserCreate}>
+      <Confirm title={title} show={show} loading={confirmLoading} onCancel={this.props.hideConfirmWindow}
+               onConfirm={function () {
+                 this.onConfirmUserCreate(options);
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: No bind in react

File: client/components/Users/UserInfoField.jsx [hunk 8]
```diff
@@ -3,8 +3,7 @@ import './UserInfoField.css';
 
 export default class UserInfoField extends Component {
   static propTypes = {
-    title: PropTypes.string.isRequired,
-    children: PropTypes.object.isRequired
+    title: PropTypes.string.isRequired
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfoField.jsx]: Where did children go? Does this not give you eslint errors?

File: client/components/Users/UserLogs.jsx [hunk 9]
```diff
@@ -49,7 +59,7 @@ export default class UserLogs extends Component {
                 </TableRow>
               );
             })
-          }
+            }
```
Comment (sandrinodimattia) [inline client/components/Users/UserLogs.jsx]: Indenting error?

File: client/containers/Configuration/Configuration.css [hunk 10]
```diff
@@ -2,7 +2,18 @@
   padding-top: 20px;
   text-align: right;
 }
+/* BASICS */
 
-.hidden {
-  display: none;
+.CodeMirror {
```
Comment (sandrinodimattia) [inline client/containers/Configuration/Configuration.css]: Undo the changes in this file. I already fixed this.

File: client/containers/Logs.jsx [hunk 11]
```diff
@@ -29,7 +29,7 @@ class LogsContainer extends Component {
     this.props.fetchLogs(this.props.logs.nextPage);
   }
 
-  createToolbar(isBottom: false) {
+  createToolbar(isBottom:false) {
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Why did you remove the space?

File: client/containers/Users/User.jsx [hunk 12]
```diff
@@ -71,12 +77,14 @@ export default connectContainer(class extends Component {
         </div>
         <div className="row user-tabs">
           <div className="col-xs-12">
-            <Tabs defaultActiveKey={1} animation={false}>
+            <Tabs defaultActiveKey={1} animation={false} id="user_info_tabs">
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: No underscores. Use dashes

File: client/selectors/getAppsForConnection.js [hunk 13]
```diff
@@ -1,14 +1,24 @@
-import { createSelector } from 'reselect';
+export default function normalizeErrorMiddleware() {
```
Comment (sandrinodimattia) [inline client/selectors/getAppsForConnection.js]: What happened here? Why is all of the code replaced with something totally different?

File: client/store/configureStore.js (hunk 1 above)
Comment (sandrinodimattia) [inline client/store/configureStore.js]: Yes, undo the changes in this file. I already fixed this.

File: client/components/Users/UserForm.jsx [hunk 14]
```diff
@@ -102,51 +103,53 @@ export default createForm('user', class extends Component {
     _.each(memberships, (a, idx) => {
       options[idx] = { value: a, label: a };
     });
-    if (options.length == 1 && !this.state.memberships) {
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Just deleting this is not ok. A better solution has to be found for this.

Comment (igrosso) [inline client/components/Users/UserForm.jsx]: I added this check in confirm handler. No just delete

File: client/components/Users/UserForm.jsx (hunk 6 above)
Comment (igrosso) [inline client/components/Users/UserForm.jsx]: fixed, but with space or not- there no lint error in my IDE

File: client/components/Users/UserDevices.jsx (hunk 5 above)
Comment (igrosso) [inline client/components/Users/UserDevices.jsx]: also fixed but no lint errors in my IDE. But in a row - they are

File: client/components/Users/UserForm.jsx (hunk 7 above)
Comment (igrosso) [inline client/components/Users/UserForm.jsx]: fixed. No lint errors in file

File: client/components/Users/UserInfoField.jsx (hunk 8 above)
Comment (igrosso) [inline client/components/Users/UserInfoField.jsx]: I removed it because in console we have a lot of errors

 Warning: Failed propType: Required prop `children` was not specified in `UserInfoField`. Check the render method of `UserInfo`.

arning: Failed propType: Invalid prop `children` of type `string` supplied to `UserInfoField`, expected `object`. Check the render method of `UserInfo`.

children could be string or number and I guess there is no need to prop.validate this attribute

File: client/containers/Configuration/Configuration.css (hunk 10 above)
Comment (igrosso) [inline client/containers/Configuration/Configuration.css]: fixed

File: client/containers/Logs.jsx (hunk 11 above)
Comment (igrosso) [inline client/containers/Logs.jsx]: fixed

File: client/containers/Users/User.jsx (hunk 12 above)
Comment (igrosso) [inline client/containers/Users/User.jsx]: fixed in all tabs
